Source/SwiftLintBuiltInRules/Rules/Idiomatic/UnusedEnumeratedRule.swift

```diff
@@ -32,16 +38,64 @@ struct UnusedEnumeratedRule: Rule {
             Example("list.enumerated().map { idx, ↓_ in idx }"),
             Example("list.enumerated().map { ↓_, elem in elem }"),
             Example("list.enumerated().forEach { print(↓$0) }"),
-            Example("list.enumerated().map { ↓$1 }")
+            Example("list.enumerated().map { ↓$1 }"),
+            Example("""
+            list.enumerated().map {
+                $1.enumerated().forEach { print(↓$1) }
```

Comment:
It feels a bit strange that the rule triggers on the identifiers (that are not missing) even though the rule is about the `enumerated()` method.

Reply:
https://github.com/realm/SwiftLint/pull/5498/files#r1552627906